v1/python-sdk/tutorials/automl-with-azureml/forecasting-backtest-single-model/assets/retrain_models.py

```diff
@@ -143,6 +153,15 @@ def run_backtest(data_input_name: str, file_name: str, experiment: Experiment):
     x_pred["actual_level"] = y_test
     x_pred["backtest_iteration"] = f"iteration_{last_training_date}"
     date_safe = RE_INVALID_SYMBOLS.sub("_", last_training_date)
+
+    # Adding the forecast quantile logic
+    if forecast_quantiles:
```

Comment:
Let us move this part to line 139 and do:
```
# By default we will have forecast quantiles of 0.5, which is our target.
if forecast_quantiles:
    if 0.5 not in forecast_quantiles:
        forecast_quantiles.append(0.5)
    fitted_model.quantiles = forecast_quantiles
 x_pred = fitted_model.forecast_quantiles(X_test)
 x_pred.rename({0.5, "predicted_level"}, axis=1, inplace=True)
```
In this case we will not need to drop columns (remove lines 141-152)

Reply:
Done. Thanks